network-upgrades/mainnet-upgrades/byzantium.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 609
+title: "Hardfork Meta: Byzantium"
+author: Alex Beregszaszi (@axic)
+type: Meta
+status: Final
+created: 2017-04-23
+requires: 100, 140, 196, 197, 198, 211, 214, 607, 649, 658
+link: https://eips.ethereum.org/EIPS/eip-609
```

Comment:
Consider naming this header field `legacy-link` or something to make it a bit more clear that this is the authoritative source going forward, and the EIPs repository is not?